src/pybind/mgr/cephadm/services/monitoring.py

```diff
@@ -423,6 +423,20 @@ def config_dashboard(self, daemon_descrs: List[DaemonDescription]) -> None:
                 service_url
             )
 
+    def pre_remove(self, daemon: DaemonDescription) -> None:
+        """
+        Called before Alertmanager is removed
+        """
+        if daemon.hostname is not None:
+            try:
+                self.mgr.check_mon_command({"prefix": "dashboard reset-alertmanager-api-host"})
+                self.mgr.check_mon_command(
+                    {"prefix": "dashboard reset-alertmanager-api-ssl-verify"}
+                )
+                logger.info("Reset the Alertmanager values to default")
```

Comment:
I think this message might come off a bit vague to a user who doesn't know what we're doing here. I feel like it should include both the fact that we're doing this since we're removing the alertmanager daemon and that we're specifically resetting dashboard alertmanager api settings. Same goes for the similar message in the prometheus pre-remove

Reply:
Updated the message in the latest commit.